eos-updater-prepare-volume/eos-updater-prepare-volume

```diff
@@ -173,6 +176,13 @@ class VolumePreparer:
 
         return runtimes
 
+    def _add_autoinstall_flatpaks(self, flatpak_collection_refs):
+        applied_actions = EosUpdaterFlatpakInstaller.determine_flatpak_ref_actions_to_check(None)
+        for filename in applied_actions:
+            for action in applied_actions[filename]:
```

Comment:
I think you can omit a hash operation on the dict key here:
```
for filename, actions in applied_actions.items():
    for action in actions:
        …
```

Reply:
But this code will change once you’ve called `euu_flatten_flatpak_ref_actions_table()` on the `applied_actions` anyway.